deltachat-jsonrpc/src/api.rs

```diff
@@ -886,6 +881,21 @@ impl CommandApi {
         Ok(chat_id.to_u32())
     }
 
+    /// Like `secure_join()`, but allows to pass a source and a UI-path.
```

Comment:
`source` and `uipath` should be documented, their purpose and why they are `u32` isn't clear when one sees this API for the first time.